src/socketio.quiver.js

```diff
@@ -617,6 +694,17 @@ QuiverSocialProvider.prototype.addFriend_ = function(servers, userId, nick, invi
  * @param {QuiverSocialProvider.server_} server
  * @private
  */
+QuiverSocialProvider.prototype.removeServer_ = function(server) {
+  var serverKey = QuiverSocialProvider.serverKey_(server);
+
+  delete this.configuration_.self.servers[serverKey];
```

Comment:
`removeServer_` is only called in one place, during Step 1 of the connection process.  Steps 2 and 3 re-populate `this.configuration_.self.servers`.  If that doesn't seem logical to you then I'm happy to change it.